google/cloud/spanner_dbapi/types.py

```diff
@@ -4,92 +4,67 @@
 # license that can be found in the LICENSE file or at
 # https://developers.google.com/open-source/licenses/bsd
 
-# Implements the types requested by the Python Database API in:
-#   https://www.python.org/dev/peps/pep-0249/#type-objects-and-constructors
+"""Implementation of the type objects and constructors according to the
+   PEP-0249 specification.
+
+   See
+   https://www.python.org/dev/peps/pep-0249/#type-objects-and-constructors
+"""
 
 import datetime
-import time
 from base64 import b64encode
 
 
-def Date(year, month, day):
-    return datetime.date(year, month, day)
-
-
-def Time(hour, minute, second):
-    return datetime.time(hour, minute, second)
-
-
-def Timestamp(year, month, day, hour, minute, second):
-    return datetime.datetime(year, month, day, hour, minute, second)
-
-
-def DateFromTicks(ticks):
-    return Date(*time.localtime(ticks)[:3])
-
-
-def TimeFromTicks(ticks):
-    return Time(*time.localtime(ticks)[3:6])
-
-
-def TimestampFromTicks(ticks):
-    return Timestamp(*time.localtime(ticks)[:6])
-
-
-def Binary(string):
-    """
-    Creates an object capable of holding a binary (long) string value.
-    """
-    return b64encode(string)
-
-
-class BINARY:
-    """
-    This object describes (long) binary columns in a database (e.g. LONG, RAW, BLOBS).
-    """
+def _time_from_ticks(ticks, tz=None):
+    """A helper method used to construct a DB-API time value.
```

Comment:
I think we should first add system tests for these types, before merging. As they are gonna be used to convert data from user types to the Spanner types, we better check how it's working in the case close to the real.

Reply:
@IlyaFaer Reverted to the previous convention, PTAL.